src/main/webapp/resources/js/pages/sequence-files/components/FastQCCharts.jsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file renders the FastQC charts component.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Col, Layout , Row, Typography} from "antd";
+import { SPACE_MD } from "../../../styles/spacing";
+import { grey1, grey4 } from "../../../styles/colors";
+import { TabPaneContent } from "../../../components/tabs/TabPaneContent";
+import { getFastQCImages } from "../../../apis/files/sequence-files";
+import { ContentLoading } from "../../../components/loader";
+import {
+  seqObjId,
+  seqFileId
+} from "../fastqc-constants";
+
+import styled from "styled-components";
+
+const StyledImage = styled.img`
+  height: 100%;
+  width: 100%;
+  border: 1px solid ${grey4};
+  padding: ${SPACE_MD};
+  border-radius: 2px;
+`;
+
+export default function FastQCCharts() {
+
+  const [loading, setLoading] = useState(true);
+  const [perBase, setPerBase] = useState("");
+  const [perSeq, setPerSeq] = useState("");
+  const [duplicationLevel, setDuplicationLevel] = useState("");
+  const [fastQCVersion, setFastQCVersion] = useState("");
+
+  useEffect(() => {
+    getFastQCImages(seqObjId, seqFileId).then(({ perbaseChart, persequenceChart, duplicationlevelChart, fastQCVersion }) => {
+      // Convert the images from byte arrays into a png images
+      setPerBase(`data:image/png;base64,${perbaseChart}`);
+      setPerSeq(`data:image/png;base64,${persequenceChart}`);
+      setDuplicationLevel(`data:image/png;base64,${duplicationlevelChart}`);
+
+      setFastQCVersion(fastQCVersion);
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <div>
+    {
+      loading ?
+        <ContentLoading message={i18n("FastQC.fetchingCharts")}/>
+        :
+        <Layout style={{paddingLeft: SPACE_MD, backgroundColor: grey1}}>
+          <TabPaneContent title={i18n("FastQC.charts")} xxl={16}>
+            <Typography.Paragraph>{i18n("FastQC.overrepresentedSequencesDescription", fastQCVersion)}</Typography.Paragraph>
+            <Row
+              gutter={[16, 16]}
+              style={{ padding: SPACE_MD }}
+            >
+              <Col span={24} xl={16} xxl={12}>
```

Comment:
This still feels like you are leaving the images pretty small at the xl  level

Reply:
They look huge on my monitors. What do you suggest to set it to?